sdk/lib/atl/atlwin.h

```diff
@@ -320,9 +320,32 @@ class CWindow
         int wndCenterHeight = wndCenterRect.bottom - wndCenterRect.top;
         int wndWidth = wndRect.right - wndRect.left;
         int wndHeight = wndRect.bottom - wndRect.top;
+        int xPos = wndCenterRect.left + ((wndCenterWidth - wndWidth + 1) >> 1);
+        int yPos = wndCenterRect.top + ((wndCenterHeight - wndHeight + 1) >> 1);
+        RECT rcWork;
+        SystemParametersInfo(SPI_GETWORKAREA, 0, &rcWork, 0);
```

Comment:
What about child windows? Should they be centered against the (global/screen) work area? Or against their parent window?

Reply:
child windows should be centered against their parent.

I am not sure if `SPI_GETWORKAREA` gives correct result for multi-monitor setups.
It might make sense to use `MonitorFromWindow` to get the correct monitor, and `GetMonitorInfo` to get the work area of that.

Reply:
@HBelusca  and  @learn-more 
 Thanks for your comments